_includes/announce-1.0.2.md

```diff
@@ -0,0 +1,141 @@
+Reactive Streams 1.0.2 is here!
+===============================
+
+We—the Reactive Streams Special Interest Group—are happy to announce the immediate availability of `Reactive Streams version 1.0.2`.
+
+When we released 1.0.1, we promised that after JDK9 had shipped, we'd release a compatibility/conversion library to seamlessly convert between the `java.util.concurrent.Flow` and the `org.reactivestreams` namespaces.
+
+Not only have we done that in this release—we have also shipped a TCK for implementations of the `java.util.concurrent.Flow` interfaces, so that they now can be verified without having to manually adapt/convert to the `org.reactivestreams` interfaces.
+
+As usual, `1.0.2` is binary, and semantically, compatible with the previous 1.0.x releases of Reactive Streams.
```

Comment:
semantically it's specifically back-wards compatible -- I guess?

Reply:
@ktoso FWIW I think it is both backwards-and-forwards compatible?